test/integration/targets/vmware_host_dns/tasks/main.yml

```diff
@@ -0,0 +1,101 @@
+# Copyright: (c) 2018, Christian Kotte <christian.kotte@gmx.de>
+# Copyright: (c) 2018, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: store the vcenter container ip
+  set_fact:
+    vcsim: "{{ lookup('env', 'vcenter_host') }}"
+
+- debug: var=vcsim
+
+- name: Wait for Flask controller to come up online
```

Comment:
Hi,

We recently did a refactoring of the functional test-suite, the goal is to be able to run it on a regular VMware environment, this instead of vcsim.
As a consequence, you should not handle vcsim in the functional test anymore, use the `prepare_vmware_roles` role instead. e.g: https://github.com/ansible/ansible/blob/devel/test/integration/targets/vmware_drs_group/tasks/main.yml

You should also ensure the resources that are created during your test are properly removed at the end.